resources/js/helpers/object.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Get a nested difference between two objects
+ * The values of source will be returned
+ *
+ * @param {Object} target
+ * @param {Object} source
+ *
+ * @returns Object
+ */
+export function objectDiff(target, source) {
```

Comment:
Should we care about any keys that only exist within `target` and not `source`? If we're doing a pure diff, I can see that being relevant. For example, if you now delete a key from your variables in the GraphqlMutation component then it won't get recognized.

Reply:
For our current usecase it's not relevant but for a full implementation as this is now a helper it could be useful.
Currently if the key is no longer available in the source it will return the key as undefined.